docs/proposals/cascading-deletion.md

```diff
@@ -171,34 +163,28 @@ type ObjectMeta struct {
 type ObjectMeta struct {
 	...
 	ParentReferences []ObjectReference
-    Orphaning bool
 }
 ```
 
 **ObjectMeta.ParentReferences**: same as in [design #1](#api-changes).
-**ObjectMeta.Orphaning**: indicates whether to orphan the children when this object is deleted.
 
 ## New components
 
-* Garbage Collector:
-  * Almost identical to the [Garbage Collector](#new-components) in design #1.
-
-* Orphaning finalizer:
-  * Could be part of the garbage collector.
-  * Watches for update events as described in [2A](#new-components-1).
-  * If `ObjectMeta.Orphaning` of the updated object is true, removes the object from the `ParentReferences` of its children.
-    * children can be found via the DAG kept by GC, or by relisting the child resource and checking the ParentReferences field.
-  * Removes the finalizer from the `ObjectMeta.Finalizers` of the parent object.
-
-## Expected race conditions
-
-* *Controller of the parent resource creates a child object after the orphaning finalizer has updated the ParentReferences of other children*: This can be solved by implementing Status.ObservedGeneration in all resources, so that before applying the orphaning logic to children, the orphaning finalizer checks Status.ObservedGeneration to ensure the controller of the parent resource has observed the deletion. Note that in design #1, the same [race condition](#expected-race-conditions) is solved by introducing the tombstone.
+* Garbage Collector: Almost identical to the [Garbage Collector](#new-components) in design #1. Some notable differences are
+  * The Garbage Processor doesn't need to remove dangling ParentReferences. The "orphan" finalizer takes care of that.
+  * The Garbage Processor sends a deletion request for an item if none of the object listed in the ParentReferences exist.
+  * Adding a fourth component, the"orphan" finalizer:
+    * Watches for update events as described in [2A](#new-components-1).
+    * Removes the object in the event from the `ParentReferences` of its children.
+      * children can be found via the DAG kept by the GC, or by relisting the child resource and checking the ParentReferences field of each potential child object.
+    * Removes the finalizer from the `ObjectMeta.Finalizers` of this object.
 
 # Comparison between design #1 and #2
 
+The main difference between the two designs is when to update the ParentReferences. In design #1, because a tombstone is created to indicate "orphaning" is desired, the updates to ParentReferences can be deferred until the deletion of the tombstone. In design #2, the updates need to be done before the parent object is deleted from the registry.
```

Comment:
s/indicate "orphaning"/indicate when "orphaning"/
s/can be deferred until the deletion of the tombstone/can occur after deletion of the resource/